licenses/spdx/spdx.json

```
@@ -501,6 +501,15 @@
                 "scheme": "SPDX"
             }
         ]
+    },
+	{
```

This line was indented with a tab instead of spaces.

```suggestion
    {
```